### ISSUE TITLE
[FE+BE]Connection of Frontend (TalktixChatUI) and Backend (TalktixChatAPI)

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#37]
- define concrete server port 4200 / backend 8000
- change the URLs of existing end points to match the defined local ports
- consider software architecture when implementing the connection

## Acceptance criteria
- [x] information from UI are successfully being sent to backend
- [x] received message should be logged
- [x] code has been tested

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by M-HRL** [comment]
@sajjadmurtaza, i have added an review for your backend changes.

@garvinkon and @sajjadmurtaza, did you talk about how the frontend should use the endpoint?

**Comment by M-HRL** [comment]
@sajjadmurtaza I did the next review.
